projects/flimnap/main.cc

```diff
@@ -112,22 +112,27 @@ namespace {
     template <typename T>
     struct observer_initializer{
         template <typename S = T>
-        typename std::enable_if<std::is_same<S, observer::Kalman<model_t>>::value, void>::type
-        initialize(S& observer) {
+        typename std::enable_if<std::is_same<typename S::observer_t, observer::Kalman<model_t>>::value, void>::type
+        initialize(S& bicycle) {
+            typename S::observer_t& observer = bicycle.observer();
+            observer.set_Q(parameters::defaultvalue::kalman::Q(observer.dt()));
+            /* Reduce steer measurement noise covariance */
```

Comment:
I've been in consistent with comment type so thanks for bringing this up and I'll make your suggested change.